### ISSUE TITLE
English Titles Are Not Displaying

### ISSUE THREAD

**Issue by aaronth07** [Drutol/MALClient#238]
Even when enabling the setting to display English titles, the English titles are not shown.

**Comment by Drutol** [comment]
Yeah, this data is missing in current api workaround. I do plan to make a substitute for it in next update though.
